src/prototype.rs

```diff
@@ -144,7 +144,38 @@ impl<'a> FirstPass<'a> {
             return self.parse_indented_code_block(ix, remaining_space);
         }
 
+
+        // HTML Blocks
+
+        // HTML blocks may have 1-3 spaces in front of them, but our scanning
```

Comment:
I suspect this isn't going to be quite right in the case where spaces can be created from a tab, i.e. `">\t<html>"`, which yields two spaces before the html tag. This is a very subtle problem, but maybe should be handled the way code blocks do it (this is what `remaining_space` is for).